nbproject/project.properties

```diff
@@ -0,0 +1,8 @@
+include.path=${php.global.include.path}
```

Comment:
No IDE specific stuff. We can't force this on people when pulling down.
We can discuss whats suits us to automate this or git submodules to link this into your personal project.